server/config.go

```diff
@@ -47,13 +47,24 @@ type MatterbuildConfig struct {
 	Repositories              []*Repository
 
 	KubeDeployJob string
+
+	PipelineTriggers map[string]*PipelineTrigger
 }
 
 type Repository struct {
 	Owner string
 	Name  string
 }
 
+type PipelineTrigger struct {
+	Description string
+	URL         string
+	Token       string
+	Reference   string
+	Variables   map[string]string
+	Users       map[string]string
```

Comment:
Users are defined as arrays, user nickname should be defined as key and id as value. We will allow only combination of user name and id at mattermost. Whenever one of them is changed, configuration need to be changed here. (User name is configurable at matterbuild).